### PR TITLE
Three more places with remote security role offerings.

### PR DESCRIPTION
**Pull request by pbarry25**
A handful more here, open to feedback, of course. Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,7 +93,9 @@ I'm looking for a full remote position (EU)! See [edoardottt.com/cv](https://www
 | InstaCart | All | Full Remote US | [Link](https://instacart.careers/current-openings/) | ? |
 | Intel | Some | Depending on the position | [Link](https://jobs.intel.com/) | ? |
 | Kaspersky | ? | Remote | [Link](https://careers.kaspersky.com) | ? |
+| Kivu | All | Full Remote US | [Link](https://kivuconsulting.com/careers/) | ? |
 | Kraken | Most Roles | Full remote | [Link](https://jobs.lever.co/kraken) | ? |
+| Kroll | Some | Full remote (Regional) | [Link](https://careers.kroll.com/en) | ? |
```
Comment (edoardottt) [inline README.md:98]: Are you sure there are full remote positions? I can't find any ..

**Review by edoardottt — COMMENTED** [review]
Thanks for your contribution!!
Just asking about a doubt

File: README.md (hunk 1 above)
Comment (pbarry25) [reply README.md:98]: Yeah, they aren't as easy to find as they could be, which is unfortunate... But they have some. :)

If you navigate to the [job listings page](https://careers.kroll.com/en/listing-page?query=*) and then select the "Working Mode" filter on the left to be "Remote" (the page is slow to load for me, you might need to give it a few seconds before the options to choose from appear), you'll see some remote security positions (clicking [here](https://careers.kroll.com/en/listing-page?query=*&workingmode=%257B67C90E92-7055-4ED4-AAEC-4F171818D867%257D) will show that listing of remote-friendly positions).

**Review by edoardottt — APPROVED** [review]
lgtm

**Comment by edoardottt** [comment]
Thanks for your contribution! @pbarry25  Appreciated

**Comment by pbarry25** [comment]
TYVM, @edoardottt! Be well! 👋

**Pull request merged**